api/pkg/apis/v1alpha1/providers/stage/http/http.go

```diff
@@ -446,6 +449,10 @@ func (i *HttpStageProvider) Process(ctx context.Context, mgrContext contexts.Man
 						}
 					}
 					if succeeded {
+						if len(data) > httpRespBodyMaxLen {
+							sLog.InfofCtx(ctx, "  P (Http Stage): wait response body too large: %d", len(data))
+							data = data[:httpRespBodyMaxLen]
+						}
 						outputs["waitBody"] = string(data) //TODO: probably not so good to assume string
```

Comment:
Is this the right place we truncate data and remove the http header? We need to distinguish the content writing in activation.status and activationData. The latter one will be used for campaign flow, we cannot modify the output from previous stages, otherwise the workflow execution will encounter error. We only want to truncate and hide the sensitive data in activation.status because activation.status has size limit and is visible when observing the activation object.